### DISCUSSION TITLE
Problem with Sitemap

### DISCUSSION THREAD

**Discussion by hautph** [mkdocs/mkdocs#3304] [Q&A]
Dear,

I'm using mkdocs version 1.4.3, I got a problem with the sitemap.xml

I'm running serve with Ip local 10.10.10.10 port 8000, I'm using Nginx Proxy Manager to public mkdocs, site access is normal, but I got a problem when accessing sitemap.xml, it's displaying

<url>
<loc>http://10.10.10.10:8000/</loc>
<lastmod>2023-07-24</lastmod>
<changefreq>daily</changefreq>
</url> 

I check the source from the site folder, the link is https://devops.vinahost.vn, and confi from mkdocs.yml is "site_url: https://devops.vinahost.vn/"

Pls help me to fix this issues

Thanks.

**Comment by ydkmlt84** [comment]
I know this is old but it is the only thing I could find with my problem. I just wanted to say I am seeing the same thing.

**Reply by agelospanagiotakis** [reply]
i am facing the same issue 
any solution on this one ?
i tryed overriding the template sitemap.xml with no success
```xml
<?xml version="1.0" encoding="UTF-8"?>
<urlset xmlns="http://www.sitemaps.org/schemas/sitemap/0.9">
{%- for file in pages -%}
    {% if not file.page.is_link and (file.page.abs_url or file.page.canonical_url) %}
    <url>
         <loc>
            {% if file.page.canonical_url %}
                {{ file.page.canonical_url|replace('0.0.0.0:8000', 'docs.mysite.gr')|e }}
            {% else %}
                {{ file.page.abs_url|replace('0.0.0.0:8000', 'docs.mysite.gr')|e }}
            {% endif %}
         </loc>
         {% if file.page.update_date %}<lastmod>{{file.page.update_date}}</lastmod>{% endif %}
    </url>
    {%- endif -%}
{% endfor %}
</urlset>

```

**Reply by ydkmlt84** [reply]
Mine is working now. I can try and share parts of my setup that may help, because I don't know what I did. Well actually, I don't self host anymore. I use a GitHub page. That was probably what did it for me.

**Comment by pawamoy** [comment]
MkDocs is a static site generator (SSG), there's no need to `serve` the site behind an NginX (reverse) proxy, just `build` then serve the static files directly with NginX 😉

**Reply by agelospanagiotakis** [reply]
ok the solution is to set in mkdocs.yml
```
extra_templates:
  - sitemap.xml
```
add docs/sitemap.xml as i posted it above
